### DISCUSSION TITLE
How to get the defer tag on encore_scripts <script

### DISCUSSION THREAD

**Discussion by neilf184** [symfony/symfony#47782] [Q&A]
I've been attempting to upgrade by webpack-encore  from 0.31.1 to 1.8.2 (wepack 4 to webpack  5) so as to be able to have the 'defer' tag included on the <script> html elements for encore_scripts. Having finally got the application to build (npm run dev) with webpack 1.8.2 I'm disappointed to see that the 'defer' tags are _not_ being generated.

I tried installing webpack-encore-bundle on top of my existing packages, as per article here [](https://symfony.com/blog/moving-script-inside-head-and-the-defer-attribute). This has created a webpack_encore.yaml file which includes "script_attributes: defer: true" but this has made do difference. Presumably this is because webpack is using my webpack.config.js file and ignoring webpack_encore.yaml.

Is there some way of configuring this in webpack.config.js, how can I get this working?

**Comment by MatTheCat** [comment]
`defer` get added if you use `encore_entry_script_tags` in your Twig template. This function comes with the WebpackEncoreBundle so you probably don’t call it since you did not install the bundle before.

**Reply by neilf184** [reply]
I get **ERROR: Unknown "encore_entry_script_tags" function** if I try to use that in my twig.

I have "symfony/webpack-encore-bundle": "^1.15" in my composer.json

**Reply by MatTheCat** [reply]
Did you follow https://github.com/symfony/webpack-encore-bundle#webpackencorebundle-symfony-integration-with-webpack-encore to install the WebpackEncoreBundle? Seems like something is missing.

**Reply by neilf184** [reply]
Sorry for the delay, holiday & illness intervened. Yes, followed all instructions.

**Reply by MatTheCat** [reply]
Please provide a reproducer then

**Reply by neilf184** [reply]
Sorry, not sure what that is/how to provide

**Reply by MatTheCat** [reply]
See https://symfony.com/doc/current/contributing/code/reproducer.html
